devdocs/settingUp.md

```diff
@@ -209,6 +210,8 @@ TBD
     deployed app and not the dev server).
     Note that GAE daily quota will be exhausted after 2-3 runs of the full test suite.
 
+4. (Optional) After installing Google Cloud SDK and deploying to your staging server, run `gcloud auth login` in the terminal of your PC and choose your google account for authentication.
```

Comment:
actually, I needed another step to choose the gae app, but maybe that's fine to omit since most people won't need to